scripts/host/k8s-install-plugins.sh

```diff
@@ -42,36 +42,3 @@ if [[ -n $PLUGINS ]]; then
     "$APP_PATH" plugin install $PLUGINS
```

Comment:
Since default plugins are shipped with image, above you can ensure we only download 'Non-go' plugins.